namespaces/live.cloud-platform.service.justice.gov.uk/laa-aws-control/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: laa-aws-control-admin
+  namespace: laa-aws-control
+subjects:
+  - kind: Group
+    name: "github:laa-aws-control"
```

Comment:
It appears that the group you are a part of is laa-aws-control-admin-team, would you be able to update this on line 8 please and the RBAC check should should be fine